components/lists/lists.html

```diff
@@ -0,0 +1,160 @@
+            </ul>
+
+        </div>
+
+
+        <div id="buttons">
+            <div class="component-flex">
+                <div class="component-info">
+                    <h1>Lists</h1>
+                    <p class="txt-m bd-btm mb-2">
+                        Various forms of Lists
+                    </p>
+                </div>
+                <div class="component-display">
+                    <h2 class="mb-1">Spaced (normal lists)</h2>
+                    <div class="display-container flex-icons mb-2">
+                        <div class="list-component ">
+                            <div class="list-title txt-s ">Category</div>
+                            <div class="divider"></div>
+                            <ul>
+                                <li class="txt-s item-hover gray "> <a href="">Home</a></li>
+                                <li class="txt-s item-hover gray"><a href="">Badminton</a></li>
+                                <li class="txt-s item-hover gray"><a href="">Training</a></li>
+                                <li class="txt-s item-hover gray"><a href="">Suppliments</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+
+                <!-- <div class="component-code">
+                    <code>
+                        <iframe src="https://www.thiscodeworks.com/embed/61fc2df66ab14d0015c32df8"
+                            style="width: 100%; height: 138px;" frameborder="0"></iframe>
+                    </code>
+                </div> -->
+
+                <div class="component-display">
+                    <h2 class="mb-1">Stacked (Notification Page)</h2>
+                    <div class="display-container  mb-2">
+                        <div class="list-component ">
+                            <div class="txt-s ">Category</div>
+                            <div class="divider"></div>
+                            <ul>
+                                <li class="txt-s indent-x item-hover gray "> <a href="">Home</a></li>
+                                <li class="txt-s gray">
+                                    <div class="indent-x">
+                                        <a class="item-hover " href="">Badminton</a>
+                                        <ul class="indent-x ">
+                                            <li class="indent-x item-hover"><a href="">Rackets</a></li>
+                                            <li class="indent-x item-hover"><a href="">Guts</a></li>
+                                            <li class="indent-x item-hover"><a href="">Shoes</a></li>
+                                        </ul>
+                                    </div>
+                                </li>
+                                <li class="txt-s indent-x item-hover gray"><a href="">Training</a></li>
+                                <li class="txt-s indent-x item-hover gray"><a href="">Suppliments</a></li>
```

Comment:
use predefined utility class instead of indent

Reply:
done